### ISSUE TITLE
Wireless debugging IP Address always changed whenever adguard turned on

### ISSUE THREAD

**Issue by cyberboh** [AdguardTeam/AdguardForAndroid#4514]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.0 Nightly 62

### Environment

```markdown
- OS:Android 13
- Device: Poco F3
- Firmware: Custom rom
```

### Root access

- [X] Yes, I have it.

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard Chinese filter, AdGuard Japanese filter, AdGuard Russian filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Widgets filter, AdGuard DNS filter, ABPindo, EasyList China, EasyList Hebrew, AdBlockID, Peter Lowe's Blocklist, List-KR, Legitimate URL Shortener

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track signals, Strip URLs of tracking parameters, Disable WebRTC, Block Push API, Block Location API

### Issue Details

Steps to reproduce:
1. Turn On adguard
2. Turn on usb debugging inside developer options
3. enable wireless debugging, the ip will be changed

### Expected Behavior

IP still using local IP, not changed

### Actual Behavior

IP has been changed to another IP, so wireless debugging can't be connected while adguard is active

### Screenshots

<details><summary>Screenshot 1 while adguard is active, i can not connect:</summary>

![SS2](https://user-images.githubusercontent.com/99109969/224320783-07115e3e-f86d-4ff0-9112-a1d5e7f9ee31.png)
![cmd](https://user-images.githubusercontent.com/99109969/224320995-198e9285-5122-4c60-aede-1db7c03a27a3.jpg)

</details>

<details><summary>Screenshot 2 while adguard is not active, i can connect:</summary>
![SS1](https://user-images.githubusercontent.com/99109969/224321161-e4a5b237-1533-4168-87fc-2c774bba0e8a.png)
![cmd 2](https://user-images.githubusercontent.com/99109969/224321199-a7ecddd4-0c8b-413c-aedb-d358b6f616f0.jpg)

**Comment by maxikuzmin** [comment]
@cyberboh yes, this is normal behavior. The IP changes because AdGuard creates a local VPN.
You can try setting the "Automatic proxy" so that the local VPN is not created and the IP is not changed

**Comment by cyberboh** [comment]
> @cyberboh yes, this is normal behavior. The IP changes because AdGuard creates a local VPN. You can try setting the "Automatic proxy" so that the local VPN is not created and the IP is not changed

where i can find automatic proxy settings?

**Comment by maxikuzmin** [comment]
@cyberboh as I understand it, you have a rooted device, the main condition is to have root access. 
Go to Settings -> Filtering -> Network -> Routing mode -> click on Automatic proxy.
This might help, let me know the result please

**Comment by maxikuzmin** [comment]
@cyberboh any news?

**Comment by cyberboh** [comment]
> @cyberboh any news?

recently tested, yea it works as intended
but for vpn type, still my ip always changed

**Comment by maxikuzmin** [comment]
@cyberboh yes, as I said earlier, it's normal behavior.The IP changes because AdGuard creates a local VPN.
Just use "Automatic proxy" so that the IP does not change.
